HQSmokeTests/testPages/users/mobile_workers_page.py

```diff
@@ -119,6 +124,11 @@ def __init__(self, driver):
         self.download_filter = (By.XPATH, "//button[@data-bind='html: buttonHTML']")
         self.error_403 = (By.XPATH, "//h1[text()='403 Forbidden']")
 
+        # add MW to group
+        self.delete_group = (By.XPATH, "//a[@href='#delete_group_modal']")
+        self.confirm_delete = (By.XPATH, "//button[@class='btn btn-danger disable-on-submit']")
+        self.delete_success_message = (By.XPATH, "//div[@class='alert alert-margin-top fade in html alert-success']")
+
```

Comment:
Are these being used anywhere?

Reply:
No, these are not being used anymore. I have removed it from the code.

Reply:
https://github.com/dimagi/dimagi-qa/commit/96d9afb02f02eb9aa7a84c6456b380300ec9097e